src/sentry/auth/authenticators/u2f.py

```diff
@@ -91,30 +90,27 @@ def _key_does_not_match(device):
     def generate_new_config(self):
         return {}
 
-    def start_enrollment(self, user, has_webauthn_register):
-        if has_webauthn_register:
-            credentials = []
-            for registeredKey in self.get_u2f_devices():
-                if type(registeredKey) == AuthenticatorData:
-                    credentials.append(registeredKey.credential_data)
-                else:
-                    c = create_credential_object(registeredKey)
-                    credentials.append(c)
-
-            registration_data, state = self.webauthn_registration_server.register_begin(
-                user={
-                    "id": user.id.to_bytes(64, byteorder="big"),
-                    "name": user.username,
-                    "displayName": user.username,
-                },
-                credentials=credentials,
-                # user_verification is where the authenticator verifies that the user is authorized
-                # to use the authenticator, this isn't needed for our usecase so set a discouraged
-                user_verification="discouraged",
-            )
-            return cbor.encode(registration_data), state
-
-        return u2f.begin_registration(self.u2f_app_id, self.get_u2f_devices()).data_for_client
+    def start_enrollment(self, user):
+        credentials = []
+        for registeredKey in self.get_u2f_devices():
+            if type(registeredKey) == AuthenticatorData:
```

Comment:
could we add a comment here describing what these two different registeredKey types are? I've already forgotten haha.